examples/topics/stats/pyramid.py

```diff
@@ -0,0 +1,57 @@
+''' A pyramid plot showing the age distribution of the male and female Titanic passengers using the `Bokeh` hbar glyph.
+
+.. bokeh-example-metadata::
+    :sampledata: titanic
+    :apis: bokeh.plotting.figure.hbar
+    :refs: :ref:`ug_topics_stats_pyramid`
+    :keywords: pyramid, hbar
+'''
+import numpy as np
+
+from bokeh.models import CustomJSTickFormatter, Label
+from bokeh.plotting import figure, show
+from bokeh.sampledata.titanic import data as df
+
+sex_group = df.groupby("sex")
+
+female_ages = sex_group.get_group("female")["age"].dropna()
+male_ages = sex_group.get_group("male")["age"].dropna()
+
+bin_width = 5
+bins = np.arange(0, 72, bin_width)
+m_hist, edges = np.histogram(male_ages, bins=bins)
+f_hist, edges = np.histogram(female_ages, bins=bins)
+
+colors = ("#055BB2", "#CB6805", "#808080")
+bar_width = 4
+
+p = figure(title="Age population pyramid of titanic passengers, by gender", height=400, width=600,
+           x_range=(-90, 90), x_axis_label="count")
+
+p.hbar(right=m_hist * -1, y=edges[1:], height=bar_width, color=colors[0], line_width=0)
```

Comment:
I don't think a separate `bar_width` is necessary, it would be best in any case to derive the bar_width from the bin_with, so that if someone experiments with the example and changes the `bin_width` the bar width can also adjust automatically 
```suggestion
p.hbar(right=m_hist * -1, y=edges[1:], height=bin_width*0.8, color=colors[0], line_width=0)
```

Reply:
Yeah, this makes sense. Thank you!